lib/src/base-styles/_nx-forms.scss

```diff
@@ -104,5 +90,9 @@
   border: 0;
   margin: 0 0 $nx-spacing-xl 0;
   padding: 0;
+
+  .nx-label {
```

Comment:
I'm not seeing any `.nx-label`s that are descendants of `.nx-fieldset`s

Reply:
> I'm not seeing any `.nx-label`s that are descendants of `.nx-fieldset`s

Removed here: https://github.com/sonatype/sonatype-react-shared-components/pull/122/commits/1c38ba52ad23e9a7a48948ca4119360923e9fc9a